WebContent/active/Unvalidated-Redirect/Redirect-FalsePositives-GET/inclusion-logic3.jsp

```diff
@@ -284,8 +285,15 @@
 										
 						if (defaultInputType == DefaultInputType.RELATIVE_INPUT) {
 							//relative to current dir path
+							MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
+							Set<ObjectName> hosts = mbs.queryNames(
+								new ObjectName("Catalina:type=Host,*"), null);
+							// There should only be one configured (default is localhost)
+							String firstHostName = (String) mbs.getAttribute(hosts.get(0), "name");
+
 							defaultBasePath = FileConstants.HTTP_PREFIX
-					       		+ request.getServerName() + ":" +
+							+ firstHostName + ":" +
+								//+ request.getServerName() + ":" +
```

Comment:
Was this formatted with some tool? Looks odd.

Reply:
No, seems like the whole codebase could use something. I'll try to adjust it.